charts/vald/templates/gateway/filter/networkpolicy.yaml

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (C) 2019-2023 vdaas.org vald team <vald@vdaas.org>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+{{- $lb := .Values.gateway.lb -}}
+{{- $filter := .Values.gateway.filter -}}
+{{- if .Values.defaults.networkPolicy.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: filter-allow
+spec:
+  podSelector:
+    matchLabels:
+      app: {{ $filter.name }}
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
+    - from:
+      - namespaceSelector:
+          matchLabels:
+            kubernetes.io/metadata.name: kube-system
+  egress:
+    # allow all the egress to communicate with user-defined filters
```

Comment:
I think we can exclude kube-system for filter request.

Reply:
[1aea701](https://github.com/vdaas/vald/pull/2022/commits/1aea701ea176adde9abb091a3d13e2cc3653b933)